### ISSUE TITLE
AttributeError in retrieving passages for MS MARCO passage ranking

### ISSUE THREAD

**Issue by kelvin-jiang** [castorini/anserini#1222]
In the replication of the MS MARCO passage ranking experiment, we retrieve passages by running
```
python ./src/main/python/msmarco/retrieve.py --hits 1000 --threads 1 \
 --index indexes/msmarco-passage/lucene-index-msmarco --qid_queries collections/msmarco-passage/queries.dev.small.tsv \
 --output runs/run.msmarco-passage.dev.small.tsv
```

This leads to the following error:
```
Traceback (most recent call last):
  File "./src/main/python/msmarco/retrieve.py", line 58, in <module>
    hits = searcher.search(query.encode('utf8'), int(args.hits))
  File "/home/kelvin/.local/lib/python3.6/site-packages/pyserini/search/pysearch.py", line 99, in search
    return self.object.search(JString(q.encode('utf8')), k)
AttributeError: 'bytes' object has no attribute 'encode'
```

This error is also encountered in https://github.com/castorini/anserini/pull/1220.

**Comment by lintool** [comment]
@kelvin-jiang thanks for reporting. yup, we're aware of this issue... the repo is in rapid flux right now due to EMNLP deadlines and TREC COVID runs. We'll return to this once things quiet down.

**Comment by lintool** [comment]
I believe this has been closed by #1238 

@kelvin-jiang can you please confirm and then close issue?

**Comment by kelvin-jiang** [comment]
Confirmed!
